thunder/core/utils.py

```diff
@@ -635,6 +635,10 @@ def _reify(x):
 #
 
 
+def is_cpu_scalar_tensor(a: Any) -> bool:
```

Comment:
This is a nice utility function (it could also expect that the input is a tensor and not do the first check, but that's fine, too)